BrowserKit/Tests/SiteImageViewTests/URLProcessing/URLCacheFileManagerTests.swift

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import XCTest
+@testable import SiteImageView
+
+class URLCacheFileManagerTests: XCTestCase {
+
+    var subject: DefaultURLCacheFileManager!
+    var mockFileManager: MockFileManager!
+
+    override func setUp() {
+        mockFileManager = MockFileManager()
+        subject = DefaultURLCacheFileManager(fileManager: mockFileManager)
+    }
+
+    override func tearDown() {
+        mockFileManager = nil
+        subject = nil
+    }
+
+    func testGetURLCache() async {
+        _ = await subject.getURLCache()
+        XCTAssertEqual(mockFileManager.fileExistsCalledCount, 1)
+        XCTAssertEqual(mockFileManager.urlsCalledCount, 2)
+    }
+
+    func testSaveURLCache() async {
+        await subject.saveURLCache(data: Data())
+        XCTAssertEqual(mockFileManager.urlsCalledCount, 1)
```

Comment:
Maybe adding a message here for expected result in those `XCTAssertEqual`, I'm not sure I understand why urlsCalledCount is 2.